### PR TITLE
Fix English in godot-4-0-sets-sail

### PR DESCRIPTION
**Pull request by JohnVeness**
Fixes various typos etc.

Note, as a native English speaker I have no idea what "withstanding issues" means! I have changed to "longstanding issues" as a guess but that may be entirely wrong. Could possibly just be changed to simply "issues"?

### CODE REVIEW

File: collections/_article/godot-4-0-sets-sail.md [hunk 1]
```diff
@@ -17,11 +17,11 @@ We're proud to work with you. Godot 4.0 is the culmination of years of your time
...
 
 Without further ado, feel free to jump straight to the [download page](/download)!
 
-Our friends at [GDQuest](https://www.gdquest.com/) prepared a wonderful video to highlight just how huge the Godot 4.0 release is. They also took the lead on writing this blog post, enabling the contributors from the Production team to focus on technical aspects of this release. The GDQuest team makes amazing [open source demos](https://github.com/gdquest-demos/godot-4.0-new-features) and [YouTube tutorials](https://www.youtube.com/@GDQuest), and they make [professional courses for Godot](https://gdquest.mavenseed.com/courses/)!
+Our friends at [GDQuest](https://www.gdquest.com/) prepared a wonderful video to highlight just how huge the Godot 4.0 release is. They also took the lead on writing this blog post, enabling the contributors from the Production team to focus on technical aspects of this release. The GDQuest team make amazing [open source demos](https://github.com/gdquest-demos/godot-4.0-new-features) and [YouTube tutorials](https://www.youtube.com/@GDQuest), and they make [professional courses for Godot](https://gdquest.mavenseed.com/courses/)!
```
Comment (YuriSizov) [inline collections/_article/godot-4-0-sets-sail.md]: Pretty sure team is singular.

File: collections/_article/godot-4-0-sets-sail.md [hunk 2]
```diff
@@ -132,7 +132,7 @@ With [Je06jm](https://github.com/Je06jm)‘s contribution, you can now also take
 
 For starters, Godot's global illumination systems have been remade from scratch.
 
-For the first time ever, Godot 4 introduces a novel real-time global illumination technique for large open worlds. **SDFGI** - or Signed Distance Field Global Illumination was created and implemented by Juan and you can learn a lot more about it [here](/article/godot-40-gets-sdf-based-real-time-global-illumination).
+For the first time ever, Godot 4 introduces a novel real-time global illumination technique for large open worlds. **SDFGI** or Signed Distance Field Global Illumination was created and implemented by Juan and you can learn a lot more about it [here](/article/godot-40-gets-sdf-based-real-time-global-illumination).
```
Comment (YuriSizov) [inline collections/_article/godot-4-0-sets-sail.md]: I think it misses the other dash instead, wrapping the clarification.

File: collections/_article/godot-4-0-sets-sail.md [hunk 3]
```diff
@@ -585,7 +585,7 @@ Going forward, we will be relying on this vastly more stable, simpler and more p
 <a id="audio"></a>
 ## Audio
 
-Sound design and music is another area that has benefited from several refinements in Godot 4.0. It also happens to be an area that requires a lot of specialized knowledge to properly support in the engine. Luckily, our contributor Ellen Poe ([ellenhp](https://github.com/ellenhp)) has exactly what it takes, and her work on Godot 4 helped fix a large amount of withstanding issues with the audio system.
+Sound design and music is another area that has benefited from several refinements in Godot 4.0. It also happens to be an area that requires a lot of specialized knowledge to properly support in the engine. Luckily, our contributor Ellen Poe ([ellenhp](https://github.com/ellenhp)) has exactly what it takes, and her work on Godot 4 helped fix a large amount of longstanding issues with the audio system.
```
Comment (YuriSizov) [inline collections/_article/godot-4-0-sets-sail.md]: Withstanding means still present, remaining. That said, I agree, it doesn't fit here.

File: collections/_article/godot-4-0-sets-sail.md (hunk 1 above)
Comment (JohnVeness) [reply collections/_article/godot-4-0-sets-sail.md]: I did wonder about that, but later in the same sentence it says "they make professional courses". It felt odd having "makes" and "make" in the same sentence. I'm not sure what's best here.

Comment (YuriSizov) [reply collections/_article/godot-4-0-sets-sail.md]: I would treat these as separate sentences, but we can also rephrase it.

File: collections/_article/godot-4-0-sets-sail.md (hunk 2 above)
Comment (JohnVeness) [reply collections/_article/godot-4-0-sets-sail.md]: Yes, I ummed and ahhed about that. I'll put the dashes in.

File: collections/_article/godot-4-0-sets-sail.md (hunk 3 above)
Comment (JohnVeness) [reply collections/_article/godot-4-0-sets-sail.md]: I'll put "remaining" then.

File: collections/_article/godot-4-0-sets-sail.md (hunk 1 above)
Comment (JohnVeness) [reply collections/_article/godot-4-0-sets-sail.md]: I'll make them separate sentences.

**Review by YuriSizov — APPROVED** [review]

**Comment by JohnVeness** [comment]
I just spotted and fixed a caption typo.

I also note that many of the pictures don't have captions at all, but I'm not volunteering to fix those!

(Sorry @YuriSizov for requesting a re-review - I wasn't sure how to make a PR no longer a draft, and pressed a hopeful looking button which accidentally summoned you! Before I found the correct button somewhere else).

File: collections/_article/godot-4-0-sets-sail.md [hunk 4]
```diff
@@ -294,7 +294,7 @@ The much-anticipated port to .NET 6 now allows users to target a newer framework
 
 Of course, the 4.0 release is also a great opportunity to break compatibility and improve the API. If you’re a C# user we highly recommend checking out this [blog post by Raul Santos](/article/whats-new-in-csharp-for-godot-4-0/), to find out all that’s new in C# for Godot 4.0.
 
-One of the most notable changes is the use of 64-bit types as scalar values. This means many APIs that used int or float now use long and double with the most noticeable being the _Process method. A Variant type is also now implemented in every API that takes variants where  System.Object was used in the past. This brings some improvements such as avoiding boxing the values.
+One of the most notable changes is the use of 64-bit types as scalar values. This means many APIs that used int or float now use long and double with the most noticeable being the _Process method. A Variant type is also now implemented in every API that takes variants where System.Object was used in the past. This brings some improvements such as avoiding boxing the values.
```
Comment (akien-mga) [inline collections/_article/godot-4-0-sets-sail.md]: Just adding this while at it (will commit myself).
```suggestion
One of the most notable changes is the use of 64-bit types as scalar values. This means many APIs that used int or float now use long and double with the most noticeable being the `_Process` method. A Variant type is also now implemented in every API that takes variants where `System.Object` was used in the past. This brings some improvements such as avoiding boxing the values.
```

**Review by akien-mga — APPROVED** [review]
Looks great, thanks!

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**